less/logos.less

```diff
@@ -32,7 +32,7 @@
 	background-image: @logo-text-inverse;
 }
 
-.flag {
+.flag, .fast-flag {
```

Comment:
Could we just deprecate `.flag`, how bad would it be? Right now I imagine we'll end up supporting both for an unknown time

Reply:
I think too bad for us to deal with right now. We can write down a ticket though.